frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx

```diff
@@ -855,10 +855,12 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = (props) => {
           {(!_.isEmpty(error) || !_.isEmpty(_.compact(_.values(formErrors)))) && (
             <Alert
               isInline
-              className="co-alert co-break-word"
+              className="co-alert co-break-word co-alert--scrollable"
               variant="danger"
-              title={error || 'Fix above errors'}
-            />
+              title="Error"
+            >
+              {error || 'Fix above errors'}
+            </Alert>
```

Comment:
These changes where dropped from a previous commit https://github.com/openshift/console/pull/2915/files#diff-06a688c00728b3590cd34d83aa619b13L857-L862 but where recently added to enable scrollable long alert messages. https://github.com/openshift/console/pull/3093